package.json

```diff
@@ -107,6 +107,7 @@
     "file-loader": "0.11.2",
     "font-awesome": "^4.5.0",
     "fs-extra": "^2.1.2",
+    "home-path": "^1.0.6",
```

Comment:
is this suppoed to be here?

Reply:
oh i see, was it breaking linux builds?

seems like there should be a package-lock.json change for it

Reply:
@diracdeltas yes - I believe it's a dependency of `electron-download`. Without this, the npm install fails on Linux. I'm not exactly sure where this broke, but you can see more details at https://jenkins.brave.com/view/laptop%20builds%20(child%20jobs)/job/browser-laptop-build-linux/536/console

Reply:
Let me update the package-lock.json too...

Reply:
@diracdeltas updated 😄 👍